### PR TITLE
xpack pipelines initialization

### PR DESCRIPTION
**Pull request by sharbuz**
## What is the problem this PR solves?

Jenkins->Buildkite pipelines migration 

X-PACK pipelines initialization:
- auditbeat
- dockerlogbeat
- filebeat
- heartbeat
- osquerybeat
- 
## Related issues

https://github.com/elastic/ingest-dev/issues/1693

### CODE REVIEW

**Review by v1v — APPROVED** [review]

File: catalog-info.yaml [hunk 1]
```diff
@@ -728,3 +728,233 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: BUILD_AND_READ
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
```
Comment (dliappis) [inline catalog-info.yaml:733]:
```suggestion
# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
```

see https://elastic.slack.com/archives/C0522G6FBNE/p1701937307239289?thread_ts=1701936859.719789&cid=C0522G6FBNE

File: catalog-info.yaml [hunk 2]
```diff
@@ -728,3 +728,233 @@ spec:
...
+      # env:
+      #   ELASTIC_PR_COMMENTS_ENABLED: "true"            #TODO: uncomment after tests
+      teams:
+        ingest-fp:
+          access_level: MANAGE_BUILD_AND_READ
+        everyone:
+          access_level: READ_ONLY
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
```
Comment (dliappis) [inline catalog-info.yaml:779]: same comment as before, please use https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json

File: catalog-info.yaml [hunk 3]
```diff
@@ -728,3 +728,233 @@ spec:
...
```
Comment (dliappis) [inline catalog-info.yaml:825]: same comment as before, please use https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json

File: catalog-info.yaml [hunk 4]
```diff
@@ -728,3 +728,233 @@ spec:
...
```
Comment (dliappis) [inline catalog-info.yaml:871]: same comment as before, please use https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json

File: catalog-info.yaml [hunk 5]
```diff
@@ -728,3 +728,233 @@ spec:
...
```
Comment (dliappis) [inline catalog-info.yaml:917]: same comment as, before please use https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json

**Review by dliappis — COMMENTED** [review]
LGTM, let's use the correct version for `rre.schema.json` in `catalog-info.yaml`

**Review by dliappis — APPROVED** [review]
LGTM

**Review by jlind23 — APPROVED** [review]

**Pull request merged**